docs/website/docs/plus/getting-started/installation.md

```diff
@@ -3,12 +3,156 @@ title: Installation
 description: Installation information for dlt+
 ---
 
-# Installing
+# Installation
 
-We distribute packages via PyPI, for example to install `dlt-plus` package:
+dlt+ requires a valid license to run, which you can obtain from dltHub by [joining our waiting list](https://info.dlthub.com/waiting-list).
+
+:::info Supported Python versions
+
+dlt+ currently supports Python versions 3.9-3.12.
+
+:::
+
+## Quickstart
+
+To install the `dlt-plus` package, run:
 
 ```sh
 pip install dlt-plus
 ```
 
-Please install a valid license before proceeding, as described below.
\ No newline at end of file
+Please install a valid license before proceeding, as described under [licensing](#licensing).
+
+## Setting up your environment
+
+### Configuration of the Python environment
+
+Check if your Python (versions 3.9-3.12) environment is configured:
```

Comment:
```suggestion
Check if your Python environment is configured:
```
I suggest to put the Python versions only to one place, so we have less places to change when the supported versions change